zephyr/esp32/src/bt/esp_bt_adapter.c

```diff
@@ -1398,3 +1402,17 @@ static void esp_bt_free(void *mem)
 {
 	esp_bt_free_func(mem);
 }
+
+esp_power_level_t zephyr_to_esp_ble_power_level(int tx_power) {
+	// Map the Zephyr power levels to the closest ESP-IDF levels
+	if (tx_power >= 9) return ESP_PWR_LVL_P9;
+	else if (tx_power >= 6) return ESP_PWR_LVL_P6;
+	else if (tx_power >= 3) return ESP_PWR_LVL_P3;
+	else if (tx_power >= 0) return ESP_PWR_LVL_N0;
+	else if (tx_power >= -3) return ESP_PWR_LVL_N3;
```

Comment:
not sure it is intended but for 2 it maps `ESP_PWR_LVL_N0`, but -2 it maps `ESP_PWR_LVL_N3`, so for negative values it round to the more negative (asymmetric?)

Reply:
-1 maps to `ESP_PWR_LVL_N3` (-1 > -3) instead of `ESP_PWR_LVL_N0`, which is closer